plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -1048,6 +1115,27 @@ yargs(process.argv.slice(2))
       process.exitCode = extract(argv.pluginPackages, argv.outputPlugins);
     }
   )
+  .command(
+    'pack <pluginPath> <outputDir>',
+    'Creates a tarball of the plugin package in the format Headlamp expects.',
```

Comment:
I wonder if it should error before overwriting? I guess it would be similar to `npm pack`, and overwrite.

Reply:
I will make it err if the destination tarball exists.